src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReportTest.java

```diff
@@ -757,6 +757,109 @@ public void testOneOrMoreFlagsUnsupported_OS() throws ParseException {
+        // first generate and add all of the facts created above
+        commands.addAll(Utils.newInsertCommands(facts));
+        // then generate the 'fireAllRules' command
+        commands.add(CommandFactory.newFireAllRules(NUMBER_OF_FIRED_RULE_KEY));
+        // add the query to retrieve the report we want
+        commands.add(CommandFactory.newQuery(QUERY_IDENTIFIER, "GetWorkloadInventoryReports"));
+
+        // execute the commands in the KIE session and get the results
+        Map<String, Object> results = Utils.executeCommandsAndGetResults(kieSession, commands);
+
+        // check that the number of rules fired is what you expect
+        Assert.assertEquals(5, results.get(NUMBER_OF_FIRED_RULE_KEY));
+        // check the names of the rules fired are what you expect
+        Utils.verifyRulesFiredNames(this.agendaEventListener,
+                // BasicFields
+                "Copy basic fields and agenda controller",
+                // Flags
+                "Flag_Hotplug_Configured",
+                // Target
+                "Target_RHV",
+                // Complexity
+                "One_Flag_Supported_OS",
+                // Workloads
+                "SsaEnabled_System_Services_Present"
+        );
+
+        // retrieve the QueryResults that was available in the working memory from the results
+        QueryResults queryResults= (QueryResults) results.get(QUERY_IDENTIFIER);
+
+        // Check that the number of object is the right one (in this case, there must be just one report)
+        Assert.assertEquals(1, queryResults.size());
+
+        // Check that the object is of the expected type and with the expected identifier (i.e. "report")
+        QueryResultsRow queryResultsRow = queryResults.iterator().next();
+        Assert.assertThat(queryResultsRow.get("report"), instanceOf(WorkloadInventoryReportModel.class));
+
+        // Check that the object has exactly the fields that the rules tested should add/change
+        WorkloadInventoryReportModel workloadInventoryReportModel = (WorkloadInventoryReportModel) queryResultsRow.get("report");
+        // BasicFields
+        Assert.assertEquals("IMS vCenter",workloadInventoryReportModel.getProvider());
+        Assert.assertEquals("V2V-DC",workloadInventoryReportModel.getDatacenter());
+        Assert.assertEquals("Cluster 1",workloadInventoryReportModel.getCluster());
+        Assert.assertEquals("vm tests",workloadInventoryReportModel.getVmName());
+        Assert.assertEquals(100000001L,workloadInventoryReportModel.getDiskSpace(), 0);
+        Assert.assertEquals(4096,workloadInventoryReportModel.getMemory().intValue());
+        Assert.assertEquals(4,workloadInventoryReportModel.getCpuCores().intValue());
+        Assert.assertEquals("Red Hat Enterprise Linux Server release 7.6 (Maipo)",workloadInventoryReportModel.getOsDescription());
+        Assert.assertEquals("rhel",workloadInventoryReportModel.getOsName());
```

Comment:
Please, add format so we have a blank space before the comma ` _,`

Reply:
done